modules/graphstudio/pages/map-data-to-graph.adoc

```diff
@@ -44,31 +44,24 @@ This section contains subsections for each data source.
 
 === Add local data file
 
-After clicking btn:[Local File], you will be prompted to choose a local file to upload through the GUI.
+After clicking btn:[Local File], you will be prompted to choose one or more local files to upload through the GUI.
 
 image::choose-a-local-file.png[]
```

Comment:
This comment is for the `choose-a-local-file.png`. Do we still have the deleted image? Because that choose a local file dialog is still shown after the user uploads the files.